<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RootWindow</class>
 <widget class="QMainWindow" name="RootWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1023</width>
    <height>768</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1023</width>
    <height>768</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>170</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>170</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>170</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>过程力检测</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="mainBtn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>660</y>
      <width>146</width>
      <height>108</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>52</red>
          <green>86</green>
          <blue>120</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>52</red>
          <green>86</green>
          <blue>120</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>52</red>
          <green>86</green>
          <blue>120</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>52</red>
          <green>86</green>
          <blue>120</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>52</red>
          <green>86</green>
          <blue>120</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>52</red>
          <green>86</green>
          <blue>120</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>52</red>
          <green>86</green>
          <blue>120</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>52</red>
          <green>86</green>
          <blue>120</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>52</red>
          <green>86</green>
          <blue>120</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 22pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>主界面</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setBtn">
    <property name="geometry">
     <rect>
      <x>146</x>
      <y>660</y>
      <width>146</width>
      <height>108</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>136</red>
          <green>136</green>
          <blue>136</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>136</red>
          <green>136</green>
          <blue>136</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>136</red>
          <green>136</green>
          <blue>136</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>136</red>
          <green>136</green>
          <blue>136</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>136</red>
          <green>136</green>
          <blue>136</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>136</red>
          <green>136</green>
          <blue>136</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>136</red>
          <green>136</green>
          <blue>136</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>136</red>
          <green>136</green>
          <blue>136</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>136</red>
          <green>136</green>
          <blue>136</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 22pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>标准设置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="historyBtn">
    <property name="geometry">
     <rect>
      <x>292</x>
      <y>660</y>
      <width>146</width>
      <height>108</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 22pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>历史记录</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sensorBtn">
    <property name="geometry">
     <rect>
      <x>438</x>
      <y>660</y>
      <width>148</width>
      <height>108</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 22pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>系统设置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="logBtn">
    <property name="geometry">
     <rect>
      <x>586</x>
      <y>660</y>
      <width>146</width>
      <height>108</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">padding:0 0px;font: 22pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>查看日志</string>
    </property>
   </widget>
   <widget class="QPushButton" name="helpBtn">
    <property name="geometry">
     <rect>
      <x>732</x>
      <y>660</y>
      <width>146</width>
      <height>108</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">padding:0 0px;margin:0 0px;
font: 22pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>帮助说明</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitBtn">
    <property name="geometry">
     <rect>
      <x>878</x>
      <y>660</y>
      <width>146</width>
      <height>108</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">padding:0 0px;margin:0 0px;
font: 22pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>退出系统</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1031</width>
      <height>661</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="rootLay"/>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
